src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
+const tableElemet = document.querySelector('table');
```

Comment:
There is a typo in the variable name `tableElemet`. It should be `tableElement` to match the intended meaning and avoid potential runtime errors.